x-pack/plugins/serverless_security/public/components/get_started/sections.tsx

```diff
@@ -0,0 +1,114 @@
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+
+import {
+  Section,
+  ProductId,
+  SectionId,
+  GetMoreFromElasticSecurityCardId,
+  GetSetUpCardId,
+  IntroductionSteps,
+} from './types';
+import * as i18n from './translations';
+import respond from './images/respond.svg';
+import protect from './images/protect.svg';
+
+export const ActiveConditions = {
+  anyCondition: [ProductId.analytics, ProductId.cloud, ProductId.endpoint],
+  analyticsToggled: [ProductId.analytics],
+  cloudToggled: [ProductId.cloud],
+  endpointToggled: [ProductId.endpoint],
+};
+
+export const sections: Section[] = [
+  {
+    id: SectionId.getSetUp,
+    title: i18n.GET_SET_UP_TITLE,
+    cards: [
+      {
+        title: i18n.INTRODUCTION_TITLE,
+        icon: { type: 'securityApp', size: 'xl' },
+        id: GetSetUpCardId.introduction,
+        activeConditions: ActiveConditions.anyCondition,
+        steps: [
+          {
+            id: IntroductionSteps.watchOverviewVideo,
+            title: i18n.WATCH_OVERVIEW_VIDEO_TITLE,
+            description: [
+              i18n.WATCH_OVERVIEW_VIDEO_DESCRIPTION1,
+              i18n.WATCH_OVERVIEW_VIDEO_DESCRIPTION2,
+              i18n.WATCH_OVERVIEW_VIDEO_DESCRIPTION3,
+            ],
+            splitPanel: (
+              <iframe
+                allowFullScreen
+                className="vidyard_iframe"
+                frameBorder="0"
+                height="100%"
+                referrerPolicy="no-referrer"
+                sandbox="allow-scripts allow-same-origin"
+                scrolling="no"
+                src="//play.vidyard.com/K6kKDBbP9SpXife9s2tHNP.html?"
+                title={i18n.WATCH_OVERVIEW_VIDEO_HEADER}
+                width="100%"
+              />
+            ),
+            timeInMinutes: 3,
+            badges: [
+              { id: 'analytics', name: i18n.PRODUCT_BADGE_ANALYTICS },
+              { id: 'cloud', name: i18n.PRODUCT_BADGE_CLOUD },
+              { id: 'edr', name: i18n.PRODUCT_BADGE_EDR },
+            ],
+          },
+        ],
+      },
+      {
+        icon: { type: 'agentApp', size: 'xl' },
+        title: i18n.BRING_IN_YOUR_DATA_TITLE,
+        id: GetSetUpCardId.bringInYourData,
+        activeConditions: ActiveConditions.anyCondition,
+      },
+      {
+        icon: { type: 'advancedSettingsApp', size: 'xl' },
+        title: i18n.ACTIVATE_AND_CREATE_RULES_TITLE,
+        id: GetSetUpCardId.activateAndCreateRules,
+        activeConditions: ActiveConditions.anyCondition,
+      },
+      {
+        icon: { type: protect, size: 'xl' },
+        title: i18n.PROTECT_YOUR_ENVIRONMENT_TITLE,
+        id: GetSetUpCardId.protectYourEnvironmentInRuntime,
+        activeConditions: [...ActiveConditions.cloudToggled, ...ActiveConditions.endpointToggled],
```

Comment:
I think It would be easier to change this `activeConditions` by `productTypeRequired` optional entry. This way we would only need to set it in this line:
`productTypeRequired: [ProductId.cloud, ProductId.endpoint],`
For the rest (`ActiveConditions.anyCondition`) it could be just omitted. The `ActiveConditions` enum won't be needed anymore.

And the check would look like this:
```
const { productTypeRequired } = cardItem;
if (!productTypeRequired || productTypeRequired.some((productType) => activeSections.has(productType))) {
   [...]
```